### DISCUSSION TITLE
Can you customize "connect outcomes to existing activity"?

### DISCUSSION THREAD

**Discussion by mrubelmann** [elsa-workflows/elsa-core#2559] [Q&A] [answered]
Hi,

I'm a newcomer to Elsa and am totally loving it!  One little issue though is that I'm finding the method to reconnect existing activities to be rather unintuitive.  Is it possible for me to override it somehow or would I need to fork and modify?  I'd love to be able to just grab the end of a line and drag it to a different box.  It would be much simpler than the two-clicks-and-a-shift thing.

Thanks for any help and for all the great work!

**Comment by sfmskywalker** [comment, marked as answer]
I am glad to hear it, thank you!

There's no way to customize the reconnect experience, but you're more than welcome to submit a PR that allows you to do it via drag & drop. I agree this would be much better.

I'm also considering backporting the Elsa 3 designer to Elsa 2, which offers a completely different designer experience, adding drag & drop support, selecting multiple activities, copy & paste, and drag & drop connections between activities on the canvas like this:

![drag-drop-connections](https://user-images.githubusercontent.com/938393/146752386-110832c5-fd50-43ab-a378-442662272c87.gif)

The designer isn't finished yet, but I hope to be able to backport it early next year.

**Reply by mrubelmann** [reply]
Wow, that looks fantastic!  Is that going to work with the Elsa 2 engine or is this just part of Elsa 3?  I see there aren't any boxes checked on the Elsa 3 roadmap yet.

**Reply by tomy2105** [reply]
Yes, this definitely looks fantastic :).
Keep my fingers crossed we'll see it in Elsa 2.x soon :).

**Reply by sfmskywalker** [reply]
We do want to backport the V3 designer once it reaches close to completion :)
